### PR TITLE
#9: write explorer files as json

### PR DESCRIPTION
**Pull request by rombautt**
Please review and merge if ok.

### CODE REVIEW

File: foundations/src/main/java/io/yields/math/framework/kpi/ExplorerDAO.java [hunk 1]
```diff
@@ -154,7 +169,61 @@ public static void save(Explorer<?> explorer) {
...
         }
+    }
+
+    private static void exportToJson(Explorer<?> explorer, File destinationFile) {
+
+        ObjectMapper jsonMapper = getObjectMapper();
+
+        try {
+
+            List<String> variableNames = explorer.all().findFirst()
```
Comment (jgheerardyn) [inline foundations/src/main/java/io/yields/math/framework/kpi/ExplorerDAO.java]: layout of this looks not too good...

File: foundations/src/main/java/io/yields/math/framework/kpi/ExplorerDAO.java [hunk 2]
```diff
@@ -154,7 +169,61 @@ public static void save(Explorer<?> explorer) {
...
+                                                                                                                    .collect(toList())).collect(toList());
+
+            List<List<Double>> statsValues = explorer.getStats().stream().map(stats -> stats.getStatsNames().stream().map(stats::getStats).collect(toList()))
+                                                     .collect(toList());
+
+            List<List<Double>> descriptorValues = explorer.getStats().stream()
+                                                          .map(stats -> stats.getDescriptorNames().stream().map(stats::getDescriptorAt).collect(toList()))
+                                                          .collect(toList());
+
+            ExplorerResult result = new ExplorerResult(variableNames, propertiesNames, statsNames, descriptorNames, variableValues, propertiesValues,
```
Comment (jgheerardyn) [inline foundations/src/main/java/io/yields/math/framework/kpi/ExplorerDAO.java]: Could you add a test to make sure that the order of the descriptor, KPI, etc. values corresponds to the order of the names?

File: foundations/src/main/java/io/yields/math/framework/kpi/ExplorerDAO.java (hunk 1 above)
Comment (rombautt) [inline foundations/src/main/java/io/yields/math/framework/kpi/ExplorerDAO.java]: fixed (format on commit by IntelliJ was the cause)

File: foundations/src/main/java/io/yields/math/framework/kpi/ExplorerDAO.java (hunk 2 above)
Comment (rombautt) [inline foundations/src/main/java/io/yields/math/framework/kpi/ExplorerDAO.java]: You're absolutely right, added test.

**Pull request merged**